frontend/src/modules/machinery/reducer.js

```diff
@@ -72,13 +92,23 @@ const initial: MachineryState = {
     entity: null,
     sourceString: '',
     translations: [],
+    searchResults: [],
+    fetching: false,
+    hasMore: false,
 };
 
 export default function reducer(
     state: MachineryState = initial,
     action: Action,
 ): MachineryState {
     switch (action.type) {
+        case CONCORDANCE_SEARCH:
+            return {
+                ...state,
+                searchResults: state.searchResults.concat(action.searchResults),
```

Comment:
Use whatever you find more intuitive, but given that object spread is already used one line above, you may write this using the array spread operator, too:
```js
searchResults: [...state.searchResults, ...action.searchResults],
```
Both `concat()` and array spread return a new array and are interchangeable.

Reply:
As with the comment below, this is another instance of following the existing pattern within the code base, and using the same format as that used for the Entities list.

Reply:
OTOH, since there's just a handful of cases like this in the codebase (or even just one?), you could change it as part of this PR if you prefer the spread operator. :trollface: